src/librustc_mir/interpret/const_eval.rs

```diff
@@ -584,17 +591,17 @@ fn numeric_intrinsic<'tcx>(
     bits: u128,
     kind: Primitive,
 ) -> EvalResult<'tcx, Scalar> {
-    let defined = match kind {
-        Primitive::Int(integer, _) => integer.size().bits() as u8,
+    let size = match kind {
+        Primitive::Int(integer, _) => integer.size(),
         _ => bug!("invalid `{}` argument: {:?}", name, bits),
     };
-    let extra = 128 - defined as u128;
+    let extra = 128 - size.bits() as u128;
     let bits_out = match name {
         "ctpop" => bits.count_ones() as u128,
         "ctlz" => bits.leading_zeros() as u128 - extra,
         "cttz" => (bits << extra).trailing_zeros() as u128 - extra,
         "bswap" => (bits << extra).swap_bytes(),
```

Comment:
Hm, so we have these duplicated between miri and CTFE? :/

Reply:
miri should just forward to ctfe for these